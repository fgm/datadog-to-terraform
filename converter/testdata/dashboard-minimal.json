{ "title": "An example screenboard which causes the issue" }
