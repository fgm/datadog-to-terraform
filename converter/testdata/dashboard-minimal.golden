resource "datadog_dashboard" "dashboard-minimal" {
title = "An example screenboard which causes the issue"
}